yarn.lock

```diff
@@ -2500,7 +2500,7 @@
   dependencies:
     "@octokit/openapi-types" "^10.0.0"
 
-"@opentripplanner/base-map@^3.0.1", "@opentripplanner/base-map@^3.0.2":
+"@opentripplanner/base-map@^3.0.1":
```

Comment:
Can this dependency be removed?

Reply:
We'll have to run the internal otp-ui dependency update again